### PR TITLE
Support for multiple majors versions

### PR DESCRIPTION
**Pull request by ShogunPanda**
This PR add supports to keep multiple majors versions in the docs.
It also refactors and simplifies the workflow by moving to just 2 Node.js scripts.

### CODE REVIEW

**Comment by ShogunPanda** [comment]
**NOTE**

Due to https://github.com/askorama/orama/issues/728, we have temporarily disable the Orama plugin.

**Review by mcollina — CHANGES_REQUESTED** [review]
doc-watcher.js is used by npm start, please restore it

File: scripts/build-versions.mjs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  const { exitCode, all } = await execa(cmd, args, { all: true, reject: false })
+
+  if (exitCode !== 0) {
+    throw new Error(
+      `${errorPrefix}: Process exited with code ${exitCode}.\n\n--- Process stdout and stderr ---\n\n${all}`
+    )
+  }
+}
+
+// TODO@PI: Support for main
```
[the remark this answers is not in the document]
Comment (ShogunPanda) [reply scripts/build-versions.mjs:24]: No, I fixed that. Will remove it.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**